src/powershell/FinOpsToolkit.psm1

```diff
@@ -0,0 +1,274 @@
+}
+
+<#
+    .SYNOPSIS
+        Retrieves available version numbers of the FinOps Hub template.
+        
+    .PARAMETER Latest
+        Will only return the latest version number of FinOps Hub template.
+        
+    .EXAMPLE
+        Get-FinOpsHubVersions
+        
+        Returns all available version numbers of FinOps Hub templates.
+        
+    .EXAMPLE
+        Get-FinOpsHubVersions -Latest
+        
+        Returns only the latest version number of the FinOps Hub templates.
+#>
+function Get-FinOpsHubVersions
+{
+    [CmdletBinding()]
+    param
+    (
+        [Parameter()]
+        [switch]
+        $Latest
+    )
+    
+    # url for testing
+    $releaseUri = 'https://api.github.com/repos/dsccommunity/xdhcpserver/releases'
+
+    #$releaseUri = 'https://api.github.com/repos/microsoft/cloud-hubs/releases'
+    $releases = (Invoke-WebRequest -Uri $releaseUri -Verbose:$false | ConvertFrom-Json)
+    
+    if ($Latest)
+    {
+        $releases = $releases | Select-Object -First 1
+        Write-Verbose -Message ($LocalizedData.FoundLatestRelease -f $releases.tag_name)
+    }
+    
+    return $release.tag_name
+}
+
+function Save-GitHubRelease
+{
+    [CmdletBinding()]
+    param
+    (
+        [Parameter(Mandatory = $true)]
+        [string]
+        $RepositoryName,
+
+        [Parameter()]
+        [string]
+        $Tag = 'latest',
+
+        [Parameter()]
+        [string]
+        $Destination
+    )
+    
+    New-Directory -Path $Destination
+
+    $lookupUri = 'https://api.github.com/repos/{0}/releases' -f $RepositoryName
+    $releases = (Invoke-WebRequest -Uri $lookupUri -Verbose:$false | ConvertFrom-Json)
+
+    if ($Tag -eq 'Latest')
+    {
+        $release = $releases | Select-Object -First 1
+        $Tag = $release.tag_name
+        Write-Verbose -Message ($LocalizedData.FoundLatestRelease -f $Tag)
+    }
+    else
+    {
+        $release = $releases | Where-Object -FilterScript {$_.tag_name -eq $Tag}
+    }
+
+    $downloadFile = $release.assets.name
+    Write-Verbose -Message ($LocalizedData.FoundAsset -f $downloadFile)
+    if ([System.IO.Path]::GetExtension($downloadFile) -eq '.nupkg')
+    {
+        $saveFileName = $downloadFile -replace '.nupkg', '.zip'
+    }
+    else
+    {
+        $saveFileName = $downloadFile
+    }
+
+    $saveFilePath = Join-Path -Path $Destination -ChildPath $saveFileName
+    $downloadUri = $release.assets.browser_download_url
+    $null = Invoke-Webrequest -Uri $downloadUri -OutFile $saveFilePath -Verbose:$false
+
+    if ([System.IO.Path]::GetExtension($saveFilePath) -eq '.zip')
+    {
+        Write-Verbose -Message ($LocalizedData.ExpandingZip -f $saveFilePath)
+        Expand-Archive -Path $saveFilePath -DestinationPath ($saveFilePath -replace '.zip', '')
+        Remove-Item -Path $saveFilePath -Recurse -Force
+    }
+}
+
+<#
+    .SYNOPSIS
+        Deploys a FinOpsHub instance.
+
+    .PARAMETER HubName
+        Name of the FinOpsHub instance.
+
+    .PARAMETER Scope
```

Comment:
Remove scope parameter. We only support RG.

Reply:
done